pipeline/lib/current_bench_json.ml

```diff
@@ -131,11 +134,20 @@ module V2 = struct
 
   let metric_of_json t =
     let units = Json.get "units" t |> Json.to_string_option |> default "" in
+    let ascending =
+      Json.get "ascending" t |> Json.to_bool_option |> default true
```

Comment:
I'm not sure that `ascending` is the most clear term, does `true` indicates that higher-is-better? (by default I think that lower-is-better is the most common type of benchmarks)

Reply:
Yes, I wasn't sure that `ascending` is the best term to use either. I took the term from a [measurement and evaluation framework](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/result/indicator/) I worked with in the past.  I'm open to other suggestions for the name. 

I haven't looked at all the benchmarks we currently have, but for some thing like the compiler, I think `lower-is-better` would be more common. 

Also, I'm not sure if we should have a default value/meaning for the existing benchmarks that have already been written. It could be confusing for users who aren't aware of the new `flag` that we are adding here?

Reply:
Yes it could be a `bool option` for when the flag is not set (or an explicit sum type to improve readability: `Unknown | Lower_is_better | Higher_is_better`... sometimes it's not clear what a bool means!)
But it seems easy enough to fix by the user if the default is not right... And we could help by using the metric to guess a better default? (a simple heuristic, as ".../s" is more probably a HiB, but otherwise benchmarks are generally LiB ?)

The word `ascending` makes sense with the documentation! I'll try to find a more explicit word just in case :)